components/homme/src/theta-l/element_ops.F90

```diff
@@ -433,13 +434,19 @@ subroutine set_thermostate(elem,ps,temperature,hvcoord)
   enddo
   elem%state%ps_v(:,:,nt)=ps
 
+
+  if (present(qv)) then
+     ! get ideal gas constant for moist air, store in variable "dp"
+     call get_R_star(dp,qv(:,:,:))
```

Comment:
Can we change the "dp" variable to "Rstar"?
As I was looking into this I keep getting confused about what's happening on these lines.